andela/aragon-cfh
# #141404601 Cards should be categorized by region

#### What does this PR do?
Adds regionalization/localization to the game.

#### Description of Task to be completed?
A player who starts a game should have the option of selecting a region for the game. This would ensure the cards are relevant to that particular geographical region.

#### How should this be manually tested?
Run the app and launch it in the browser.
When starting a new game, you should be given the option to select a region for the game.
The app should also attempt to automatically locate what region of the world you're in.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#141404601

#### Questions:
None for now
